Mathlib/SetTheory/Game/Impartial.lean

```diff
@@ -24,25 +24,28 @@ open scoped PGame
 
 namespace PGame
 
-/-- The definition for an impartial game, defined using Conway induction. -/
-def ImpartialAux (G : PGame) : Prop :=
+private def ImpartialAux (G : PGame) : Prop :=
   (G ≈ -G) ∧ (∀ i, ImpartialAux (G.moveLeft i)) ∧ ∀ j, ImpartialAux (G.moveRight j)
 termination_by G
 
-theorem impartialAux_def {G : PGame} : G.ImpartialAux ↔
-    (G ≈ -G) ∧ (∀ i, ImpartialAux (G.moveLeft i)) ∧ ∀ j, ImpartialAux (G.moveRight j) := by
-  rw [ImpartialAux]
+/-- An impartial game is one that's equivalent to its negative, such that each left and right move
+is also impartial.
+
+Note that this is a slightly more general definition than the one that's usually in the literature,
+as we don't require `G ≡ -G`. Despite this, the Sprague-Grundy theorem still holds: see
+`SetTheory.PGame.equiv_nim_grundyValue`.
 
-/-- A typeclass on impartial games. -/
+In such a game, both players have the same payoffs at any given moment. -/
```

Comment:
Subposition might be what you're looking for here: (I believe, at least in Aaron Siegel's book, that
a non-proper subposition also includes the game itself.)

```suggestion
In such a game, both players have the same payoffs at any subposition. -/
```

Reply:
I like that word, I'll take it!